tasks/agent.py

```diff
@@ -534,6 +534,24 @@ def omnibus_build(
     print("Omnibus: {}".format(omnibus_elapsed))
 
 
+@task
+def build_dep_tree(
+    ctx, tag="",
```

Comment:
Maybe reword as `git_ref`?
```suggestion
    ctx, git_ref="",
```

You can actually pass any git reference to `git checkout` and I can see someone wanting to switch to a branch/commit instead of a tag.